democracy/tests/test_section_poll.py

```diff
@@ -246,6 +255,7 @@ def test_post_section_poll_answer_multiple_choice_second_answers(john_doe_api_cl
     assert poll.n_answers == 1
 
 
+@pytest.mark.skipif(isArchLinux, reason="Weird bug in http requests")
```

Comment:
This needs to figure out more and get a proper reason or reference for the issue or solution. Commit message should contain more information about this issue. It could have affects to other distros as well.

Reply:
http get/post request response order seems reversed on arch based distros (happened with manjaro and vanilla arch), reason unknown

`AssertionError: assert {'answers': [2, 3], 'question': 1, 'type': 'multiple-choice'} in [{'answers': [3, 2], 'question': 1, 'type': 'multiple-choice'}, {'answers': [5], 'question': 2, 'type': 'single-choice'}]`

Reply:
I consulted our Linux specialist and he said that this could be a distro problem, but he said that this is very exceptional. He suggests that the problem should find out first from the application-level before making a distro-specific hacking.

Reply:
Agreed that this needs better commit message which contains the information about the issue, debugging, references, testing etc. about so the next developers know more about this hack.

Reply:
Comments about the issue added in commit 30ac8fc34619f05261f9036d1524b84d496955f7

Reply:
This comment was intended to add to exist commit message https://github.com/City-of-Turku/kerrokantasi/pull/15/commits/4b0b7c3ad4c85a4785e523a6411e97c6d6a6d8c2, but why not also to code. With Git Blame/Annotate functionality this can be easily checked from the code row that who has made the change, when and why.

Reply:
This is fine now. I recommend reading Frank's guiding which was in frontend's PR. https://github.com/City-of-Turku/kerrokantasi-ui/pull/29#pullrequestreview-466469133